static/src/javascripts/projects/commercial/modules/dfp/lazy-load.ts

```diff
@@ -43,7 +45,11 @@ const getObserver = once(() =>
 );
 
 export const enableLazyLoad = (advert: Advert): void => {
-	if (dfpEnv.lazyLoadObserve) {
+	const useGptLazyLoad = isInVariantSynchronous(
+		commercialGptLazyLoad,
+		'variant',
+	);
+	if (dfpEnv.lazyLoadObserve && !useGptLazyLoad) {
```

Comment:
For readability could we extract this condition to something like `useCustomIntersectionObserver`?

```suggestion
	const useCustomLazyLoad = dfpEnv.lazyLoadObserve && !useGptLazyLoad;
	if (useCustomLazyLoad) {
```